homework4/ParseTree/ParseTree/Operand.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace ParseTree;
+
+/// <summary>
+///     Class for operands.
+/// </summary>
+public class Operand : INode
+{
+    /// <summary>
+    ///     Initializes operand node.
+    /// </summary>
+    /// <param name="value">value of operand node.</param>
+    public Operand(int value)
+    {
+        Value = value;
+    }
+
+    /// <summary>
+    ///     Gets and sets operand's value.
+    /// </summary>
+    public float Value { get; set; }
```

Comment:
```suggestion
    public float Value { get; private set; }
```
Разве предполагается, что `Value` будут менять извне?